test/test_core_aten_ops.py

```diff
@@ -29,6 +29,7 @@ def diff_output(testcase, output1, output2, rtol, atol, equal_nan=True):
     output2_cpu = output2.detach().cpu()
     if output2_cpu.dtype != output1.dtype:
       output2_cpu = output2_cpu.to(output1.dtype)
+    # import pdb; pdb.set_trace()
```

Comment:
nit: remove debugging code

Reply:
Let's leave it there... I always put it back for debugging for every single op worked. So its easier to uncomment for the next time. Lets remove after all ops are done XD